dask/dataframe/core.py

```diff
@@ -3137,16 +3148,24 @@ def value_counts(
                 )
             kwargs["dropna"] = dropna
 
+        meta = self._meta.value_counts(normalize=normalize)
+
+        if split_out > 1 and normalize:
+            aggregate_kwargs["length"] = (
+                len(self) if dropna is False else len(self.dropna())
```

Comment:
I'm not sure if this is acceptable. Basically I needed a way to pass the total len into the split_out combine.

Reply:
This seems reasonable to me - I guess an "optimal" solution for multi-partition results would be to collect/aggregate/combine the counts within the first aca pass.  However, I'm not sure it is completely worth the additional code complexity to do this, because the result would need to be quite large to benefit (and split_out is often 1 anyway).